README.md

```
@@ -1,11 +1,29 @@
-Minimal starter code for a ktor project
-
-- Uses Netty as the server engine
-- Uses yaml for declarative configuration
+This is an example project demonstrating how to manage ktor authentication with an exposed database
 
 ## Running
+
+### Generate a hex secret for HMAC SHA-256
+
+Run the following command to generate a hex secret for HMAC SHA-256
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)